src/supertux/debug.hpp

```diff
@@ -39,6 +39,8 @@ class Debug
   // vaguely measure the impact of code changes which should increase the FPS
   bool draw_redundant_frames;
 
+  bool show_tile_ids_in_editor;
```

Comment:
Maybe `show_toolbox_tile_ids` would be a better name, because it's shorter, and also mentions the toolbox specifically?